src/gtfs/read.rs

```diff
@@ -1416,4 +1378,10 @@ mod tests {
             );
         });
     }
+
+    fn extract_ids<T: Id<T>>(c: &Collection<T>) -> Vec<String> {
+        let mut ids: Vec<String> = c.values().map(|l| l.id().to_string()).collect();
+        ids.sort();
+        ids
+    }
```

Comment:
@TeXitoi 
with this function I cannot get line ids from routes
I can do 
```rust
fn extract_ids<T: Id<T> + Id<U>, U: Id<U>>(c: &Collection<T>) -> Vec<String> {
    let mut ids: Vec<String> = c.values()
        .map(|l| <T as Id<U>>::id(l).to_string())
        .collect();
    ids.sort();
    ids
}

extract_ids::<Route, Line>(&collections.routes)
```
What do you think?